src/components/WalletInit/RestoreWallet/ImportReadOnlyWalletScreen.js

```diff
@@ -0,0 +1,148 @@
+/* eslint-disable react-native/no-inline-styles */
+// @flow
+import React from 'react'
+import {compose} from 'redux'
+import {View, ScrollView, Dimensions} from 'react-native'
+import {withHandlers} from 'recompose'
+import {injectIntl, defineMessages, type intlShape} from 'react-intl'
+import QRCodeScanner from 'react-native-qrcode-scanner'
+import DeviceInfo from 'react-native-device-info'
+
+import {Text, BulletPointItem} from '../../UiKit'
+import {CONFIG} from '../../../config/config'
+import {WALLET_INIT_ROUTES} from '../../../RoutesList'
+import {withNavigationTitle, onDidMount} from '../../../utils/renderUtils'
+import {Logger} from '../../../utils/logging'
+import {errorMessages} from '../../../i18n/global-messages'
+import {showErrorDialog} from '../../../actions'
+import {
+  isValidPublicKey,
+  isCIP1852AccountPath,
+} from '../../../utils/bip44Validators'
+
+import styles from './styles/ImportReadOnlyWalletScreen.style'
+
+import type {ComponentType} from 'react'
+import type {Navigation} from '../../../types/navigation'
+
+const messages = defineMessages({
+  title: {
+    id: 'components.walletinit.importreadonlywalletscreen.title',
+    defaultMessage: '!!!Read-only Wallet',
+  },
+  paragraph: {
+    id: 'components.walletinit.importreadonlywalletscreen.paragraph',
+    defaultMessage:
+      '!!!To import a read-only wallet from the Yoroi ' +
+      'extension, you will need to:',
+  },
+  line1: {
+    id: 'components.walletinit.importreadonlywalletscreen.line1',
+    defaultMessage: '!!!Open "My wallets" page in the Yoroi extension.',
+  },
+  line2: {
+    id: 'components.walletinit.importreadonlywalletscreen.line2',
+    defaultMessage:
+      '!!!Click on the QR code icon of the wallet you want to ' +
```

Comment:
As a reminder, I don't think we will use a QR code icon for this button in the Yoroi Extension since we still want in the future a way to export the wallet as file or export the private key (which would probably be from the same menu)